packages/terra-core-docs/CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # Changelog
 
 ## Unreleased
+* Added
+  * Added visual label and associated with input field using 'for' and 'id'.
```

Comment:
```suggestion

* Added
  * Added visual label and associated with input field using 'for' and 'id'.
```
Need a line break to keep it consistent with other logs

Reply:
Done.